src/annotator/sidebar.coffee

```diff
@@ -36,6 +36,14 @@ module.exports = class Sidebar extends Host
 
     if @plugins.Toolbar?
       @toolbarWidth = parseInt(window.getComputedStyle(this.plugins.Toolbar.toolbar[0]).width)
+      if config.disableToolbarMinimizeBtn
+        @toolbar.find('[name=sidebar-toggle]').remove();
+      if config.disabledToolbarHighlightsBtn
+        @toolbar.find('[name=highlight-visibility]').remove();
+      if config.disableToolbarNewNoteBtn
+        @toolbar.find('[name=insert-comment]').remove();
+      if config.disableToolbarCloseBtn
+        @toolbar.find('[name=sidebar-close]').remove();
```

Comment:
As discussed on the call, the DOM structure of the toolbar is an implementation detail of the toolbar module. Rather than manipulating the DOM here, the toolbar should just expose appropriate methods.